### PR TITLE
fix(#56): update variable handling

### PR DESCRIPTION
**Pull request by zapfire88**
CSP can now handle snake case representation id.

### CODE REVIEW

File: src/manifests/handlers/dash/segment.ts [hunk 1]
```diff
@@ -37,26 +37,34 @@ export default async function dashSegmentHandler(
     const urlSearchParams = new URLSearchParams(event.queryStringParameters);
     const pathStem = path.basename(event.path).replace('.mp4', '');
     // Get the number part after "segment_"
-    const [, reqSegmentIndexStr, representationIdStr, bitrateStr, timeStr] =
+    // eslint-disable-next-line prefer-const
```
Comment (bwallberg) [inline src/manifests/handlers/dash/segment.ts:40]: I think this eslint disable can bbe removed? right? ( I saw you had `let` before )

**Review by bwallberg — APPROVED** [review]
Except the eslint disable LGTM!

File: src/manifests/handlers/dash/segment.ts (hunk 1 above)
Comment (zapfire88) [reply src/manifests/handlers/dash/segment.ts:40]: Very true! Removing it!

**Review by birme — APPROVED** [review]
LGTM

**Pull request merged**